test/Conversion/amd/buffer_load_to_local_to_llvm.mlir

```diff
@@ -0,0 +1,161 @@
+// RUN: triton-opt %s -split-input-file --allocate-shared-memory --convert-triton-amdgpu-to-llvm=arch=gfx950 | FileCheck %s --check-prefix=GFX950
+// RUN: triton-opt %s -split-input-file --allocate-shared-memory --convert-triton-amdgpu-to-llvm=arch=gfx942 --verify-diagnostics | FileCheck %s
+
+#blocked = #ttg.blocked<{sizePerThread = [1, 1], threadsPerWarp = [1, 64], warpsPerCTA = [4, 1], order = [1, 0]}>
+#shared = #ttg.swizzled_shared<{vec = 1, perPhase = 1, maxPhase = 1, order = [1, 0]}>
+#smem = #ttg.shared_memory
+module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 4 : i32, ttg.shared = 8192 : i32, ttg.target = "hip:gfx942", "ttg.threads-per-warp" = 64 : i32} {
+  // CHECK-LABEL: buffer_load_to_local_simple
+  tt.func public @buffer_load_to_local_simple(%arg0: !tt.ptr<f16> {tt.divisibility = 16 : i32, tt.pointer_range = 32 : i32},
+                                %arg1: !tt.ptr<f16>,
+                                %arg2: tensor<32x64xi32, #blocked>,
+                                %arg3: !ttg.memdesc<32x64xf16, #shared, #smem, mutable>) {
+    // Each thread needs to load 8 elements and we load 1 (sizePerThread) per buffer load instruction
+    // CHECK: rocdl.make.buffer.rsrc %arg1
```

Comment:
Why are we not checking `GFX950`? If you want this check to be both on, need something like
```
// RUN: ... --check-prefix=COMMON,GFX950
// RUN: ... --check-prefix=COMMON,GFX942
```
and then using `COMMON:` here would enable it for both.